linux_os/guide/system/auditing/authconfig_config_files_symlinks/oval/shared.xml

```diff
@@ -0,0 +1,43 @@
+<def-group>
+    <definition class="compliance" id="{{{ rule_id }}}" version="1">
+        {{{ oval_metadata("Prevent overwriting of custom authentication configuration settings by the authconfig utility") }}}
+        <criteria operator="AND" comment="password-auth and system-auth are symlinks">
+            <criterion comment="/etc/pam.d/password-auth is a symlink and points to /etc/pam.d/password-auth-local"
+                test_ref="test_password-auth_is_a_symlink" />
+            <criterion comment="/etc/pam.d/system-auth is a symlink and points to /etc/pam.d/system-auth-local"
+                test_ref="test_system-auth_is_a_symlink" />
+        </criteria>
+    </definition>
+
+    <unix:symlink_test check="all" check_existence="all_exist"
+      comment="/etc/pam.d/password-auth is a symlink and points to /etc/pam.d/password-auth-local"
+      id="test_password-auth_is_a_symlink" version="1">
+         <unix:object object_ref="object_password-auth_is_a_symlink" />
+         <unix:state state_ref="state_password-auth_is_a_symlink" />
+    </unix:symlink_test>
+
+    <unix:symlink_test check="all" check_existence="all_exist"
+      comment="/etc/pam.d/system-auth is a symlink and points to /etc/pam.d/system-auth-local"
+      id="test_system-auth_is_a_symlink" version="1">
+         <unix:object object_ref="object_system-auth_is_a_symlink" />
+         <unix:state state_ref="state_system-auth_is_a_symlink" />
+    </unix:symlink_test>
+
+    <unix:symlink_object comment="/etc/pam.d/system-auth is a symlink" id="object_system-auth_is_a_symlink" version="1">
```

Comment:
Wherever possible, could you ensure lines no longer than 99 columns, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#all-files

Reply:
Addressed